api/catalog/api/views/media_views.py

```diff
@@ -143,24 +152,50 @@ def _get_user_ip(request):
         return ip
 
     @staticmethod
-    def _get_proxied_image(image_url, width=settings.THUMBNAIL_WIDTH_PX):
-        if width is None:  # full size
-            proxy_upstream = f"{settings.THUMBNAIL_PROXY_URL}/{image_url}"
-        else:
-            proxy_upstream = (
-                f"{settings.THUMBNAIL_PROXY_URL}/"
-                f"{settings.THUMBNAIL_WIDTH_PX},fit/"
-                f"{image_url}"
-            )
+    def _thumbnail_proxy_comm(path: str, params: dict):
+        proxy_url = settings.THUMBNAIL_PROXY_URL
+        query_string = urlencode(params)
+        upstream_url = f"{proxy_url}/{path}?{query_string}"
+        log.info(f"Upstream URL: {upstream_url}")
```

Comment:
Same here:
```suggestion
        log.debug(f"Upstream URL: {upstream_url}")
```

I'll stop with the suggestion, but I think most if not all of the `log.info` calls should be at the `debug` level. Otherwise we're going to get a TON of log messages per request.

Reply:
Makes sense, I've combined the log statements and changed to `debug` in c76f1f2.